### ISSUE TITLE
Salesforce: SFDC Node

### ISSUE THREAD

**Issue by sharmmoh1983** [KumologicaHQ/kumologica-support#14]
Two enhancements recommended for the node :
`````
1. Access Token refresh functionality to be available .
2. Parametrized SOQL functionality to be available .

`````

**Comment by ab73863** [comment]
@sharmmoh1983 . Taking up the feature and will post you with the update.

**Comment by ab73863** [comment]
@sharmmoh1983 access token is now available as part of the 1.0.4 version of salesforce node.
